src/sentry/api/endpoints/organization_member_index.py

```diff
@@ -19,9 +29,27 @@ class MemberPermission(OrganizationPermission):
     }
 
 
+class OrganizationMemberSerializer(serializers.Serializer):
+    email = serializers.EmailField(max_length=75, required=True)
+    role = serializers.ChoiceField(choices=roles.get_choices(), required=True)
+    teams = ListField(required=False, allow_null=False)
+
+
 class OrganizationMemberIndexEndpoint(OrganizationEndpoint):
     permission_classes = (MemberPermission, )
 
+    @transaction.atomic
+    def save_team_assignments(self, organization_member, teams):
```

Comment:
I'd say then if we're allowing `teams` to be an empty list, maybe we just make that behavior in here explicit.

I understand that if teams is empty, `bulk_create` will do nothing. But that's kinda implicitly happening. I'd change it to:

```python
if teams:
    OragnizationMemberTeam.objects.bulk_create(...)
```

Just to clearly indicate that we're consciously expecting this behavior. Semantics are the same, but this better shows intent imo.